### PR TITLE
[Feature][Task Plugin] Enable users to switch endpoints in zeppelin tasks

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

- Enable users to switch endpoints in zeppelin tasks
- related: #9814 

## Brief change log

- Remove zeppelin endpoint config from `common.properties`.
- Enable users to configure zeppelin endpoint in zeppelin tasks to trigger jobs from different zeppelin instances.
- Update related docs.

## Verify this pull request

- Verified by UT and manual tests.

### CODE REVIEW

File: dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java [hunk 1]
```diff
@@ -32,6 +32,7 @@ public class ZeppelinParameters extends AbstractParameters {
      */
     private String noteId;
     private String paragraphId;
+    private String restEndpoint;
```
Comment (zhuangchong) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java:35]: `restEndpoint` parameter should be required

Comment (EricGao888) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java:35]: Sure, I will add a check here 
```java
    @Override
    public boolean checkParameters() {
        return StringUtils.isNotEmpty(this.noteId);
    }
```
Thx~

File: docs/docs/en/guide/resource/configuration.md [hunk 2]
```diff
@@ -121,9 +121,6 @@ development.state=false
 # rpc port
 alert.rpc.port=50052
 
-# Url endpoint for zeppelin RESTful API
-zeppelin.rest.url=http://localhost:8080
```
Comment (EricGao888) [inline docs/docs/en/guide/resource/configuration.md:125]: Also need to update this in Chinese docs.

File: dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java (hunk 1 above)
Comment (EricGao888) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-zeppelin/src/main/java/org/apache/dolphinscheduler/plugin/task/zeppelin/ZeppelinParameters.java:35]: @zhuangchong  Done, PTAL when available, thx

**Review by zhuangchong — APPROVED** [review]
+1

**Pull request merged**